test/unit/localization/initTestI18n.js

```diff
@@ -0,0 +1,28 @@
+import {init} from 'i18next';
```

Comment:
Couple thoughts here:

* Probably not discoverable enough, but generally I’ve been keeping `test/unit` to just the actual test files, and putting helpers like this in `test/helpers`
* It would be nice to avoid the side-effect of globally configuring `i18next` in the test environment—I think we can set up an isolated instance using [`i18next.createInstance()`](http://i18next.com/docs/api/#create-instance), and then make it the default export of this module?